src/objects/zcl_abapgit_object_vcls.clas.abap

```diff
@@ -5,17 +5,52 @@ CLASS zcl_abapgit_object_vcls DEFINITION PUBLIC INHERITING FROM zcl_abapgit_obje
 
   PROTECTED SECTION.
   PRIVATE SECTION.
+
 * See include MTOBJCON:
-    CONSTANTS: c_cluster_type TYPE c VALUE 'C'.
-    CONSTANTS: c_mode_insert  TYPE obj_para-maint_mode VALUE 'I'.
+    CONSTANTS c_cluster_type TYPE c VALUE 'C' ##NO_TEXT.
+    CONSTANTS c_mode_insert TYPE obj_para-maint_mode VALUE 'I' ##NO_TEXT.
 
+    CLASS-METHODS check_lock
```

Comment:
There's a method exists_a_lock_entry_for for reading enques in zcl_abapgit_objects_super.
https://github.com/larshp/abapGit/blob/06616580eb500d94d1d6ca165638c7c4ac01cf61/src/objects/zcl_abapgit_objects_super.clas.abap#L187-L212

Maybe you can use that method instead of locking and releasing?

Reply:
I think this is possible. I will adjust the section.